src/apps/orders/static/admin/check_partial_refunds.js

```diff
@@ -0,0 +1,16 @@
+(function($) {
+    $(document).ready(() => {
+        var noPartialRefundAvailable = ['dolyame', 'zero_price'];
+        var bankId = $('#id_bank_id').val();
+        var refundsGroup = $('#refunds-group');
+
+        if (noPartialRefundAvailable.includes(bankId)) {
+          setTimeout(() => {
+            django.jQuery('.add-row').hide();
```

Comment:
```suggestion
            $('.add-row').hide();
```
Мы передаём в функцию инстанс jquery, которую вызываем во всех других местах. А здесь почему-то обращаемся к глобальной jquery. Кажется это не имеет смысла.